pkg/api/persistentvolume/util_test.go

```diff
@@ -27,7 +28,7 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
-func TestDropDisabledFields(t *testing.T) {
+func TestDropDisabledFieldsCSI(t *testing.T) {
```

Comment:
this was also testing block mode... revert the name change?

Reply:
done